build-logic/publishing/src/main/kotlin/build-logic.publish-to-tmp-maven-repo.gradle.kts

```diff
@@ -14,7 +14,7 @@ val localRepoElements by configurations.creating {
     }
 }
 
-val localRepoDir = layout.buildDirectory.dir("local-maven-repo")
+val localRepoDir = project.rootProject.layout.buildDirectory.dir("local-maven-repo")
```

Comment:
Configuring a different project is often a bad idea.
If you want to create "local-maven-repo" in root project, then `publish-to-tmp-maven-repo.gradle.kts` should be applied to that project.

Could you clarify why do you want moving `local-maven-repo` to the root?

Reply:
I wanted one local-maven-repo that had both the sigstore-java and sigstore-maven-plugin published to it.